brendenwest/cascade
# Add ios

## Description
Adding the ios folder for cross-platform compatibility

## To Test
To ensure the app continues to function, test it in the cli with the
`npx react-native start`
and
`npx react-native run-android`
commands
